eng/pipelines/templates/stages/archetype-net-release.yml

```diff
@@ -77,6 +77,11 @@ stages:
                             PackageName: ${{artifact.name}}
                             ServiceName: ${{parameters.ServiceDirectory}}
                             ForRelease: true
+                        - template: /eng/common/pipelines/templates/steps/verify-restapi-spec-location.yml
+                          parameters:
+                            PackageName: ${{artifact.name}}
+                            ServiceDirectory: ${{parameters.ServiceDirectory}}
+                            GitHubSpecRepoPat: $(azure-rest-api-specs-repo-gh-pat)
```

Comment:
@weshaggard is there any existed github PAT I can leverage here?

Reply:
Do we need a PAT? I would expect no but if we do need one we probably would use `azuresdk-github-pat`

Reply:
Use the recommended token.